docs/qute/TemplateSupport.md

```diff
@@ -112,12 +112,24 @@ CodeLens are clickable to jump to the proper Java type, method or field.
 
 ## Inlay Hint for Qute
 
+All inlay hints can be turned off by disabling `qute.inlayHints.enabled` in settings.
+
+### Show Section parameter type
+
 Qute support displays the Java type of the parameter names as an inlay hint.
 
-![Inlay Hints for Qute](images/QuteTemplateInlayHint.gif)
+![Show Section parameter type](images/QuteTemplateInlayHint_ShowSectionParameterType.gif)
+
+To remove inlay hints on section parameter types only, disable `qute.inlayHints.showSectionParameterType` in settings.
+
+### Show Section parameter default value
+
+Qute support displays the default value of the parameter names as an inlay hint when user tag defines a default value (ex: `{#let name?="main.css"}`)
```

Comment:
Add a note that `qute.inlayHints.showSectionParameterType` also needs to be enabled for this to work

Reply:
It is a bug!

Reply:
Please retry it it should work now

Reply:
Good catch!

Reply:
I still need `qute.inlayHints.showSectionParameterType` enabled for the default value inlay hint to appear. Is this expected?

Reply:
You should have that :

![QuteInlayHintSettings](https://github.com/redhat-developer/vscode-quarkus/assets/1932211/08bc1836-de9b-4f69-87b6-23f5dc392626)

Reply:
I'll try recompiling everything again, but it's not working for me.

Reply:
Okay, it works now!